docs/cloud-native-security/kspm-benchmark-rules.asciidoc

```diff
@@ -11,21 +16,21 @@ Each benchmark rule checks to see if a specific type of resource is configured a
 * `Ensure IAM policies that allow full "*:*" administrative privileges are not attached`
 * `Ensure the default namespace is not in use`
 
+When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Security Posture dashboard>>. 
 
-When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Security Posture dashboard>>.
+NOTE: Benchmark rules are not editable.
 
-To find the Benchmark Integrations page, go to **Rules -> Benchmark rules**. From there, you can view the benchmark rules associated with an existing integration by clicking the integration name.
+[discrete]
+== Review your benchmarks
 
-[role="screenshot"]
-image::images/benchmark-rules.png[Benchmark rules page]
+To see your active benchmarks, go to **Rules -> Benchmarks**. From there, you can click an existing security posture management integration's name to view the benchmark rules associated with it. You can click on an individual benchmark rule to see details including information about how to remediate it, and related links. 
 
-You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
+Benchmark rules are enabled by default, but you can disable some of them to customize an integration for your environment. To enable or disable a rule, use the **Enabled** toggle on the right of the rules table. 
```

Comment:
> customize an integration for your environment

A bit similar to the previous comment, we do not provide per integration customization but rather per benchmark. so if you have 2 AWS integrations for example, and you mute a rule from AWS, it will be muted for both integrations